plugin/web/extensions/location/index.html

```diff
@@ -0,0 +1,28 @@
+<title></title>
+<style>
+    html, body{
+        background: transparent;
+    }
+    .root{
+        background: #fff;
+        border-radius: 0 0 8px 0;
+    }
+    html, body, .root {
+        width: 1680px;
+        height: 40px;
+        transition: height 0.5s, width 0.5s;
```

Comment:
Do you need this? I think this is from the Sidebar, right? Please remove anything that you don't need if you copy/paste anything.

Reply:
Done

Reply:
ca838bb63dff7bc9efdcc1e7ad0376fefc3c3404